pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -119,5 +130,43 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 	}
 }
 
+// minorUpdateFrom returns the version that was installed completed if a minor level upgrade is in progress
+// and the empty string otherwise
+func minorUpdateFrom(status configv1.ClusterVersionStatus, currentVersion semver.Version) string {
+	completedVersionStr := GetCurrentVersion(status.History)
```

Comment:
nit:
```suggestion
	completedVersion := GetCurrentVersion(status.History)
```

we do not seem to have a `completedVersion` around so the hungarian notation is not necessary (it is rarely necessary :D )

Reply:
Very true. Modified.